### PR TITLE
Shrink uri data

### PR DESCRIPTION
**Pull request by kasei**
Alleviate the memory pressure during the pipeline post-processing phase of rewriting tag URIs to UUIDs.

### CODE REVIEW

File: tests/test_sales_modeling_private_contract.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def verify_sales(self, output):
 		hmo = objects[hmo_key]
 		
 		prov_entry_curr = activities['tag:getty.edu,2019:digital:pipeline:REPLACE-WITH-UUID:sales#PROV,B-267,1817,0001']
-		prov_entry_prev = activities['tag:getty.edu,2019:digital:pipeline:REPLACE-WITH-UUID:sales#OBJ,B-267,0001,1817-seller-0-ProvenanceEntry']
+		prov_entry_prev = activities['tag:getty.edu,2019:digital:pipeline:REPLACE-WITH-UUID:sales#OBJ,B-267,0001,1817-seller-0-Prov']
```
Comment (azaroth42) [inline tests/test_sales_modeling_private_contract.py]: Wouldn't shortening the prefix be more/also effective?  

tag:getty.edu,2019:dig:pipeline-uuid:[project]#[suffix]  ?

Comment (kasei) [reply tests/test_sales_modeling_private_contract.py]: The prefix isn't stored in the URI to UUID mapping table. It's a map of `suffix -> UUID urn`.

Comment (azaroth42) [reply tests/test_sales_modeling_private_contract.py]: Ahh, so takes up no space. Gotcha

Comment (kasei) [reply tests/test_sales_modeling_private_contract.py]: There's a bit of low-hanging fruit on the rewriting process in that I could make it a map from `suffix -> UUID data` (e.g. not the full string for a UUID urn, but a shorter form that stores just the UUID data), but that adds complication to the rewriting process. Possible, but these shortenings of the suffix strings are easy and don't require any other code changes.

**Pull request merged**